skyportal/api_description.md

```diff
@@ -34,23 +37,72 @@ if response.status_code in (200, 400):
 curl -s -H 'Authorization: token ea70a5f0-b321-43c6-96a1-b2de225e0339' http://localhost:5000/api/sysinfo
 ```
 
-#### Using query parameters
+### Request parameters
 
-```python
-import requests
+There are two ways to pass information along with a request: path and body parameters.
+
+#### Path parameters
+
+Path parameters (also called query or URL parameters) are embedded in
+the URL called. For example, you can specify `numPerPage` or
+`pageNumber` path parameters when calling `/api/candidates` as
+follows:
+
+```shell
+curl -s -H 'Authorization: token ea70a5f0-b321-43c6-96a1-b2de225e0339' \
+     http://localhost:5000/api/candidates?numPerPage=100&pageNumber=1
+```
 
+When using Python's `requests` library, a dictionary of path
+parameters can be passed in via the `params` keyword argument:
+
+```python
 token = 'ea70a5f0-b321-43c6-96a1-b2de225e0339'
 
 response = requests.get(
-    'http://localhost:5000/api/candidates?numPerPage=100&pageNumber=1',
-    headers={'Authorization': f'token {token}'}
+    "http://localhost:5000/api/sources",
+    params={"includeComments": True, includeThumbnails: False},
```

Comment:
```suggestion
    params={"includeComments": True, "includeThumbnails": False},
```

Reply:
How'd this happen? Ah, @stefanv must've cloned my branch before this was fixed, and then filed a PR using that